### PR TITLE
Alerts page

### PR DESCRIPTION
**Pull request by L-Steinmacher**
I created a new component, DashboardAlerts, and added it to the Admin sider.  One function to combine  all locations into one   object with the value of the number of times the location appears.  This is for the flags in the alerts side bar.  The flags have the functionality to redirect to another component but that is not set up yet.

### CODE REVIEW

File: src/components/pages/DashboardAlerts/AlertsDummyData.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
+export const dummyData = [
+  {
+    clubid: 1,
```
Comment (ashtilawat23) [inline src/components/pages/DashboardAlerts/AlertsDummyData.js:3]: Maybe add a comment to the code to understand what the dummyData represents.

**Review by ashtilawat23 — APPROVED** [review]

File: src/components/pages/DashboardAlerts/AlertsDummyData.js (hunk 1 above)
Comment (L-Steinmacher) [reply src/components/pages/DashboardAlerts/AlertsDummyData.js:3]: I'll add that right now!

File: src/components/pages/DashboardAlerts/RenderDashboardAlerts.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  };
+
+  let flagObj = combineFlags(dummyData);
+
+  return (
+    <>
+      <div className="alerts container">
+        <h2 className="alerts header"> Alerts</h2>
+        {Object.keys(flagObj).map(location => (
+          <div className="alert div" key={location}>
```
Comment (eet349) [inline src/components/pages/DashboardAlerts/RenderDashboardAlerts.js:30]: Good job adding the key.  For some reason this is being missed a lot in this repo.

**Review by eet349 — APPROVED** [review]
Nice comments. Looks good.

**Pull request merged**